src/pages/en/guides/deploy/edgio.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Deploy your Astro Site to Edgio
+description: How to deploy your Astro site to the web using Edgio.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy your Astro project to [Edgio](https://www.edg.io/), an edge and CDN platform to deploy, protect and accelerate websites and APIs.
+
+:::tip
+Check out [the Astro guide in Edgio’s docs](https://docs.edg.io/guides/astro)!
+:::
+
+## Install Edgio CLI
+
+```bash
+npm install -g @edgio/cli
+```
+
+## Add Edgio to your Astro site
+
+```bash
+edgio init
+```
+
+## (Optional) Enable Server Side Rendering
+
+After you’ve setup [Server Side Rendering with Astro](https://docs.astro.build/en/guides/server-side-rendering/), specify server file path in edgio.config.js as below:
+
+```diff
++ import { join } from 'path'
+
+module.exports = {
++  astro: {
++    // The path of the standalone server that runs Astro SSR
++    // The dependencies for this file are automatically bundled
++    appPath: join(process.cwd(), 'dist', 'server', 'entry.mjs'),
++  },
+}
+```
```

Comment:
We have some fancy syntax highlighting tools in this repo, so we can get a nicer display like this:

````suggestion
```js ins={2,5-9}
// edgio.config.js
import { join } from 'path'

module.exports = {
  astro: {
    // The path of the standalone server that runs Astro SSR.
    // The dependencies for this file are automatically bundled.
    appPath: join(process.cwd(), 'dist', 'server', 'entry.mjs'),
  },
};
```
````

- Using the `js` language and our custom `ins` tag, we can highlight the insertions without losing syntax highlighting like you do when using the `diff` tag.
- Including the filename comment will get transformed to a pretty filename label on the page.

Reply:
Woah that's ❤️